tests/net/lib/http_server/core/src/main.c

```diff
@@ -46,183 +48,265 @@ static const unsigned char request_get_2_streams[] = {
 	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 };
 
+static const uint8_t request_get_static_simple[] = {
+	/* Magic */
+	0x50, 0x52, 0x49, 0x20, 0x2a, 0x20, 0x48, 0x54, 0x54, 0x50, 0x2f, 0x32,
+	0x2e, 0x30, 0x0d, 0x0a, 0x0d, 0x0a, 0x53, 0x4d, 0x0d, 0x0a, 0x0d, 0x0a,
+	/* SETTINGS[0] */
+	0x00, 0x00, 0x0c, 0x04, 0x00, 0x00, 0x00, 0x00,	0x00,
+	0x00, 0x03, 0x00, 0x00, 0x00, 0x64, 0x00, 0x04, 0x00, 0x00, 0xff, 0xff,
+	/* HEADERS[1]: GET / */
+	0x00, 0x00, 0x21, 0x01, 0x05, 0x00, 0x00, 0x00, TEST_STREAM_ID_1,
+	0x82, 0x84, 0x86, 0x41, 0x8a, 0x0b, 0xe2, 0x5c, 0x0b, 0x89, 0x70, 0xdc,
+	0x78, 0x0f, 0x03, 0x53, 0x03, 0x2a, 0x2f, 0x2a, 0x90, 0x7a, 0x8a, 0xaa,
+	0x69, 0xd2, 0x9a, 0xc4, 0xc0, 0x57, 0x68, 0x0b, 0x83,
+	/* SETTINGS[0] */
+	0x00, 0x00, 0x00, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00,
+	/*  GOAWAY[0] */
+	0x00, 0x00, 0x08, 0x07, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+};
+
+#define TEST_STATIC_PAYLOAD "Hello, World!"
+
 static uint16_t test_http_service_port = SERVER_PORT;
 HTTP_SERVICE_DEFINE(test_http_service, SERVER_IPV4_ADDR,
 		    &test_http_service_port, 1, 10, NULL);
 
-static const char index_html_gz[] = "Hello, World!";
+static const char index_html_gz[] = TEST_STATIC_PAYLOAD;
```

Comment:
Just to not confuse the reader, perhaps the variable should be named plain `index_html` as the data is not gzipped.

Reply:
True, I've simply renamed this and related variables to `static_resource_*` to avoid confusion in tests.